app/azure-storage/application-eventstore-repository.js

```diff
@@ -1,9 +1,11 @@
 const queryEntitiesByPartitionKey = require('./query-entities')
+const { odata } = require('@azure/data-tables')
 
 const getApplicationEvents = async (reference) => {
   const eventRecords = await queryEntitiesByPartitionKey(
     'ahwreventstore',
-    reference
+    reference,
+    odata`PartitionKey ge ${reference} and PartitionKey lt ${(+reference + 1).toString()}`

```

Comment:
could we leave a code comment about the partition key implications regarding `sbi_cph` and maybe change `reference` to `sbi` to make it explicit?

Reply:
Updates pushed with code comments added and reference changed to sbi